pkg/common/cns-lib/volume/manager.go

```diff
@@ -255,19 +270,39 @@ func ClearTaskInfoObjects() {
 					snapshotTaskMapLock.Lock()
 					defer snapshotTaskMapLock.Unlock()
 					delete(snapshotTaskMap, snapshotTaskKey)
+					if managerInstance.improvedTasksListViewEnabled {
+						err := managerInstance.taskViewManager.RemoveTaskFromListView(context.Background(), taskDetails.task.Reference())
+						if err != nil {
+							log.Errorf("failed to remove expired task %v from listview", taskDetails.task.Reference())
+						} else {
+							log.Infof("removed expired task %v from listview", taskDetails.task.Reference())
+						}
+					}
 				}()
 			}
 		}
 	}
 }
 
+func ClearInvalidTasksFromListView() {
+	ticker := time.NewTicker(time.Duration(1) * time.Minute)
```

Comment:
What is the rationale for choosing 1-minute time duration?

Reply:
I don't have a reason for it. This goroutine is supposed to remove tasks that were failed to remove from the listview and the internal map in a previous attempt. The scenario for such a failure is most likely a vpxd crash. I don't know if vpxd crashing is a frequent event. My guess is that it is not. How do I go about deciding a correct timeout duration?

Reply:
1 minute interval is too aggressive. Can you refer time interval we are using for other clean-up routines?

Reply:
The existing `ClearTaskInfoObjects()` has a 1 minute timer. 

I've set it to 5 minutes here.